test/check-loopback

```diff
@@ -42,7 +54,8 @@ class TestLoopback(MachineCase):
         b.wait_visible('#login-fatal')
         self.assertIn("The server refused to authenticate 'admin' using password authentication, and no other supported authentication methods are available.", b.text('#login-fatal'));
 
-        self.allow_journal_messages("cockpit-polkit: couldn't respond to polkit daemon: GDBus\\.Error:org\\.freedesktop\\.PolicyKit1\\.Error\\.Failed: No session for cookie")
+        self.allow_journal_messages("cockpit-polkit: couldn't respond to polkit daemon: GDBus\\.Error:org\\.freedesktop\\.PolicyKit1\\.Error\\.Failed: No session for cookie",
+                                    ".*server offered unsupported authentication methods: public-key.*")
```

Comment:
>> it has offered the password method, 

> I don't think that's true, the test disables password authentication, 

You are right.  This message does however also happen in other cases where password authentication has not been disabled but cockpit-ws doesn't have a password.

I guess I am just confused about what the problem codes authentication-not-supported, authentication-failed, and no-forwarding mean exactly.

Would it make sense to remove authentication-not-supported and no-forwarding completely (by returning authentication-failed instead) and just use the more explicit auth-methods-results member?